k8s-daemonset/k8s/namerd.yml

```diff
@@ -24,7 +24,6 @@ data:
 
     storage:
       kind: io.l5d.k8s
-      experimental: true
```

Comment:
@adleong when running this example I get:
```
namerd 1.0.0-rc1 (rev=8ffdb5e77ca98e9c164f1d0bf6613628b5779cf0) built at 20170328-112058
io.buoyant.namerd.NamerdConfig$$anon$1: The io.buoyant.namerd.storage.K8sConfig storage is experimental and must be explicitly enabled by setting the `experimental' parameter to true.
```
I thought k8s storage was graduated, am I mistaken here?